app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, foreign_key: 'post'
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the Rails Active Callbacks `after_save` to call the `update_comments_counter` method every time a new post has been saved to the database, then make the `update_comments_counter` method private